searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.hpp

```diff
@@ -113,7 +113,8 @@ template <typename DataT>
 void
 PostingListSearchContextT<DataT>::fetchPostings(bool strict)
 {
-    assert(!_fetchPostingsDone);
+    if (_fetchPostingsDone) return;
```

Comment:
why are we fetching postings more than once?

Reply:
In a conformance test.

Reply:
maybe fix the test instead?

Reply:
Done